portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,37 +33,56 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private List<String> quotes;
-    private ArrayList<String> messages;
-    @Override
-    public void init() {
-        quotes = new ArrayList<>();
-        quotes.add("The elevator to success is out of order. You'll have to use the stairs, one step at a time.");
-        quotes.add("I always wanted to be somebody, but now I realise I should have been more specific.");
-        quotes.add("I am so clever that sometimes I don't understand a single word of what I am saying");
-        quotes.add("People say nothing is impossible, but I do nothing every day");
-        quotes.add("You can't have everything. Where would you put it?");
-        quotes.add("Well-behaved women seldom make history.");
-    }
-    public ArrayList create_messages() {
-        messages = new ArrayList<>();
-        messages.add("I am a message.");
-        messages.add("Wow that last message is so self aware.");
-        messages.add("Wait, you all can talk?");
-        return messages;
-    }
-    public String toJson(ArrayList<String> alist) {
-        Gson gson = new Gson();
-        String json = gson.toJson(alist);
-        return json;
-    }
-
   @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      
+      //HTML Form Steps
+      String first = getParameter(request, "fname", "");
+      String last = getParameter(request, "lname", "");
+      String firstLast = String.format("%s %s",first,last);
+      String comment = getParameter(request, "comment", "");
+
+      HashMap<String, String> name_comment = new HashMap<String, String>();
+      name_comment.put(firstLast, comment);
```

Comment:
In Java, the official Google style is to use `camelCase` rather than `snake_case`.
https://google.github.io/styleguide/javaguide.html#s5.3-camel-case